### PR TITLE
imtoken wallet added + new browser wallet icon

### PR DESCRIPTION
**Pull request by clementbalestrat**

### CODE REVIEW

**Review by andytcf — APPROVED** [review]

File: containers/Connector/config.ts [hunk 1]
```diff
@@ -51,26 +51,24 @@ export const initOnboard = (network: Network, subscriptions: Subscriptions) => {
...
 					rpcUrl: infuraRpc,
+					preferred: true,
 				},
 				{
 					walletName: 'walletConnect',
 					rpc: { [network.id]: infuraRpc },
 					preferred: true,
 				},
-				{ walletName: 'coinbase', preferred: true },
+				{ walletName: 'imToken', rpcUrl: infuraRpc, preferred: true },
```
Comment (JChiaramonte7) [inline containers/Connector/config.ts:61]: I'm not seeing imToken wallet in our connect wallet model?
image

Comment (clementbalestrat) [reply containers/Connector/config.ts:61]: @JChiaramonte7  yeah all good it's mobile only

File: sections/exchange/hooks/useBalancerExchange.tsx [hunk 2]
```diff
@@ -254,6 +254,7 @@ const useBalancerExchange = ({
 		) {
 			const maxNoPools = 2;
 			const sor = new SOR(
+				// @ts-ignore
```
Comment (clementbalestrat) [inline sections/exchange/hooks/useBalancerExchange.tsx:257]: @JChiaramonte7 have you looked at this issue?

**Review by vbstreetz — APPROVED** [review]
tested. working well 👍🏽

**Pull request merged**